tools/machine-learning/mujoco/README.md

```diff
@@ -0,0 +1,18 @@
+# Setup
+
+Use [uv](https://docs.astral.sh/uv/).
+After installing uv, run `uv sync` to install all dependencies or directly execute an example from below.
+
```

Comment:
See the other comment about my error with glfw.

```suggestion
Make sure `glfw` is installed on your machine. For any arch-based system:
- `$ pacman -S glfw`

```
Maybe you could also add the package installation for fedora.

Reply:
For me on fedora it seems to be preinstalled. I modified the readme, I don't think it's necessary to add it to the dependencies in the documentation, since it's only needed for mujoco